__tests__/forces/Gravity_test.re

```diff
@@ -74,5 +95,77 @@ describe("Gravity", () => {
         |> toBeCloseTo(-0.0000714892571)
       )
     );
+
+    it("should respect user-supplied values of G (x value)", () =>
+      Expect.(
+        expect(
+          Gravity.gravityForceV(
+            ~attractorMass,
+            ~moverMass,
+            ~attractor,
+            ~mover,
+            ~g=1.,
+            (),
+          )
+          |> fst,
+        )
+        |> toBeCloseTo(-0.2404926216)
+      )
+    );
+
+    it("should respect user-supplied values of G (y value)", () =>
+      Expect.(
+        expect(
+          Gravity.gravityForceV(
+            ~attractorMass,
+            ~moverMass,
+            ~attractor,
+            ~mover,
+            ~g=1.,
+            (),
+          )
+          |> snd,
+        )
+        |> toBeCloseTo(-0.6413136576)
+      )
+    );
+
+    it(
```

Comment:
Adding additional tests for untested functionality. For those curious, the `threshold` parameter limits the magnitude of the gravitational force by supplying a range within the force is applied. If the `mover` exceeds the upper bound of the `threshold` parameter (i.e. upper bound is 40 and the `mover` is 50m away from the `attractor`), we supply a force equal to the `mover` being at 40m away. This is how we ensure the `mover` never goes flying off the screen. If this parameter isn't specified you can get behavior where the `mover` slingshots off quite a ways 😂